diesel/src/macros/associations/belongs_to.rs

```diff
@@ -45,15 +45,25 @@
 ///
 /// [custom_derive]: https://crates.io/crates/custom_derive
 ///
-/// ```ignore
+/// ```no_run
+/// # #[macro_use] extern crate diesel;
+/// # #[macro_use] extern crate custom_derive;
+/// # table! { users { id -> Integer, } }
+/// # table! { posts { id -> Integer, user_id -> Integer, } }
+/// # custom_derive!{
+/// #     #[derive(Identifiable(users))]
+/// #     pub struct User {
+/// #         id: i32,
+/// #     }
+/// # }
 /// custom_derive! {
-///     #[derive(BelongsTo(User, foreign_key = user_id)]
-///     #[table_name(posts)]
+///     #[derive(BelongsTo(User, foreign_key = user_id, table = posts), Identifiable(posts))]
```

Comment:
This commit doesn't change anything for other methods to invoke this macro, it only fixes the usage with custom_derive. 
It should also be possible to provide this outside of diesel, but I think it belongs into the main crate, because all other macros suggests the usage of custom_derive.

Obviously things will change as soon as macros 1.1 will become usable on stable, but up to this date custom_derive seems like the best method to provide support for stable.